.teamcity/DFastBETests/latexmanualgeneration.kt

```diff
@@ -0,0 +1,103 @@
+import jetbrains.buildServer.configs.kotlin.*
+import jetbrains.buildServer.configs.kotlin.buildFeatures.commitStatusPublisher
+import jetbrains.buildServer.configs.kotlin.buildFeatures.swabra
+import jetbrains.buildServer.configs.kotlin.buildSteps.script
+import jetbrains.buildServer.configs.kotlin.failureConditions.BuildFailureOnText
+import jetbrains.buildServer.configs.kotlin.failureConditions.failOnText
+import jetbrains.buildServer.configs.kotlin.triggers.vcs
+
+object LatexManualGeneration : BuildType({
+    id("LatexManualGeneration")
+    name = "Latex Manual Generation"
+
+    artifactRules = """
+        +:docs/end-user-docs/*.pdf
```

Comment:
can we move this `docs/end-user-docs` to a parameter or an environment variable ?